gradio/blocks.py

```diff
@@ -819,6 +821,11 @@ def set_event_trigger(
         elif every:
             raise ValueError("Cannot set a value for `every` without a `fn`.")
 
+        if _targets[0][1] == "change" and trigger_mode == None:
+            trigger_mode = "always_last"
+        elif trigger_mode == None:
+            trigger_mode = "once"
```

Comment:
Nit: we could also add a check here to see if someone passes in a trigger_mode that is not "once", "multiple", or "always_last" and if so, raise a `ValueError` (right now, it silently fails)